lintcmd/cmd.go

```diff
@@ -217,7 +217,7 @@ func success(allowedChecks map[string]bool, res runner.ResultData) []problem {
 }
 
 func filterIgnored(problems []problem, res runner.ResultData, allowedAnalyzers map[string]bool) ([]problem, error) {
-	couldveMatched := func(ig *lineIgnore) bool {
+	couldHaveMatched := func(ig *lineIgnore) bool {
```

Comment:
This code is fairly verbose, so I don't see the value in abbreviating this.

I left `could've` as the `'` makes it a word...

I'm happy to drop any commit(s) as requested...